Balloonduino.h

```diff
@@ -155,13 +133,21 @@ struct IMUData_s
 // environmental data
 struct ENVData_s
 {
+#ifndef BALLOONDUINO_NO_BME
```

Comment:
does this compile? in read_env I believe you set these fields to either the measured value (if BALLOONDUINO_NO_BME isn't defined), or zero if it is. So this field has to exist in either case.